computation/src/main/java/com/powsybl/computation/DefaultExecutionReport.java

```diff
@@ -22,29 +27,22 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultExecutionReport.class);
 
-    private static final ExecutionReport OK = new DefaultExecutionReport();
+    private final Path workingDirectory;
 
     private final List<ExecutionError> errors;
 
-    /**
-     * An execution report with no execution error.
-     * @return an execution report with no execution error.
-     */
-    public static ExecutionReport ok() {
-        return OK;
-    }
-
     /**
      * Create an execution report with no execution error.
      */
-    public DefaultExecutionReport() {
-        this(Collections.emptyList());
+    public DefaultExecutionReport(Path workingDirectory) {
+        this(workingDirectory, Collections.emptyList());
```

Comment:
Is it relevant to make a constructor without parameters with a default directory (maybe not)(don't forget the Breaking change label if it is not the case)?

Reply:
Not sure it's relevant to have a default constructor. The working directory is the temporary directory used during the execution. From my point of view, computations always run in a temporary directory, even if no data are stored in it, it's created by the computation manager (part of the API).

Reply:
Okay!